pkg/monitoring/metrics/cdi-controller/metrics.go

```diff
@@ -15,8 +15,3 @@ func SetupMetrics() error {
 		dataVolumeMetrics,
 	)
 }
-
-// ListMetrics registered prometheus metrics
-func ListMetrics() []operatormetrics.Metric {
-	return operatormetrics.ListMetrics()
-}
```

Comment:
@machadovilaca, @arnongilboa is removing all of those because they all do the same: `return operatormetrics.ListMetrics()`. 
Why did we do it in the first place, instead of calling it directly?